test/spec/modules/kargoBidAdapter_spec.js

```diff
@@ -142,6 +142,10 @@ describe('kargo adapter tests', function () {
                 model: 'model',
                 source: 1,
               }
+            },
+            site: {
```

Comment:
The `site` object can get pretty large. Should we specifically cherry-pick just the fields we need (`cat`)? Using the `prebid-testing` repo, I added the following to my Prebid.js publisher config...
```
site: {
    id: '123',
    name: 'Test Site',
    cat: ['IAB1', 'IAB2', 'IAB3']
}
```

And here's what Prebid.js sends in the `site` object...

<img width="499" alt="image" src="https://github.com/KargoGlobal/Prebid.js/assets/5454953/2b0db011-0bca-4d77-8a81-a88c5807e930">

That will definitely beef up the request and duplicate some objects (like page URL). Some publishers may have large `site` objects that will become costly from a data transfer and marshaling perspective.

Reply:
@jsadwith Yes, this is a good call out. I guess if we are at all concerned about request size we should just take site.cat. 

I will make changes now

Reply:
Changes made